docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/entity-name-is-not-unique.md

```diff
@@ -99,8 +99,7 @@ NotUnique:TransferProperty Transfer property "customProperty" for "LocaleCmsPage
 
 ## Database table name is not unique
 
-Database table names must be unique. Check the examples below to learn how to fix the error.
-
+Database table names must be unique for the new tables on the project level. Check the examples below to learn how to fix the error.
```

Comment:
It should be simplified I think. Something like:
```
On the project level all new tables in database must have unique names.
```

And I am not sure that someone says "Check the example", because it means "Validate the example". It is possible to say "Have a look on example...", but it is very informal. Here it is better to say something like:

```
Below you can find the code examples that show how to solve the described problem.
```

Reply:
Thanks ++

Reply:
Replaced with the suggested structure